README.md

```diff
@@ -93,9 +93,22 @@ Intercom.registerIdentifiedUser({ userId: 'bob' });
 ```javascript
 Intercom.registerIdentifiedUser({ userId: 'bob' })
 Intercom.updateUser({
-		email: 'email',
-		name: 'name',
-	});
+    // Pre-defined user attributes
+    // (this attributes will be added in custom_attributes as duplicate)
```

Comment:
Why do you want to duplicate attributes in custom attributes? This could cause some problems. You mentioned in the description that iOS does it this way, but I don't think it does.

Reply:
I commented my thought above, we now understand that user attributes should be placed in the root object and custom attributes should be placed in other separately name-spaced objects (`custom_attributes`). So, it seems that only thing we should care is its backward compatability.

Reply:
👍 as I mentioned above I don't think backwards compatibility is important here because the previous behaviour simply does not work.